cmd/metrics/metrics.go

```diff
@@ -134,6 +136,7 @@ func main() {
 	fHosts := flag.Int("h", 1, "No of hosts : Sample hosts required (default 1).")
 	fPlugins := flag.Int("p", 100, "No of plugins: Sample plugins per host(default 100).")
 	fIterations := flag.Int("t", 1, "No of times to run sample data (default 1) -1 for ever.")
+	fUniqueName := flag.String("uname", "metrics-"+strconv.Itoa(rand.Intn(100)), "Unique name across application")
```

Comment:
This isn't really unique since you could technically have a duplicate here right?

Reply:
You are correct.   The default value has an N/100 chance of not being unique if the default value is used.  (Where N is the number of parallel metrics consumers).  However,  when used in the intended context of Kubernetes, the value will be unique as Pod Names are unique.

Reply:
:+1: